modules/slack-configuring-acs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-slack.adoc
+:_module-type: PROCEDURE
+[id="slack-configuring-acs_{context}"]
+= Configuring {product-title}
+
+Create a new integration in {product-title} by using the Webhook URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (add icon).
```

Comment:
I'd recommend removing `(add icon)` for safety here, or at least commenting it out.
Same comment for lines 16 and 17 as well.

Reply:
Now that I understand this, I'd suggest putting backticks around the icon text (`add`) for clarity that it is a literal.